ext/standard/basic_functions.stub.php

```diff
@@ -79,6 +79,179 @@ function sort(array &$arg, int $sort_flags = SORT_REGULAR): bool {}
 
 function rsort(array &$arg, int $sort_flags = SORT_REGULAR): bool {}
 
+function usort(array &$arg, callable $cmp_function): bool {}
+
+function uasort(array &$arg, callable $cmp_function): bool {}
+
+function uksort(array &$arg, callable $cmp_function): bool {}
+
+function end(array &$arg) {}
+
+/** @param array|object $arg */
+function prev(&$arg) {}
+
+/** @param array|object $arg */
+function next(&$arg) {}
+
+function reset(array &$arg) {}
+
+/** @param array|object $arg */
+function current($arg) {}
+
+/** @return int|string|null */
+function key(array $arg) {}
+
+function min($arg, ...$args) {}
+
+function max($arg, ...$args) {}
+
+/** @param array|object $input */
+function array_walk(&$input, callable $funcname, $userdata = null): bool {}
+
+/** @param array|object $input */
+function array_walk_recursive(&$input, callable $funcname, $userdata = null): bool {}
+
+function in_array($needle, array $haystack, bool $strict = false): bool {}
+
+/** @return int|string|false */
+function array_search($needle, array $haystack, bool $strict = false) {}
+
+/** @prefer-ref $arg */
+function extract(array &$arg, $extract_type = EXTR_OVERWRITE, string $prefix = null): ?int {}
```

Comment:
```suggestion
function extract(array &$arg, $extract_type = EXTR_OVERWRITE, string $prefix = UNKNOWN): ?int {}
```

Or possibly `""`, if that's really equivalent to not specifying the prefix.